README.rst

```diff
@@ -0,0 +1,46 @@
+
+CellML Specification
+====================
+
+This repository will be used by the CellML editors to develop the CellML specification (starting with version 1.2). The content has been reworked to include an informal guide embedded within the normative specification text (in version 2.0).
+
+If things work correctly, changes in this repository will be reflected over at: https://cellml-specification.readthedocs.org/.
+
+Building the documentation locally
+----------------------------------
+
+The documentation can be built on your local machine following these instructions. The following instructions are suitable for Unicies derived operating systems and may need modification for other operating systems.
+
+First clone a local copy of the CellML specification repository::
+
+  git clone https://github.com/cellml/cellml-specification.git
+
+Next setup a virtual environment::
+
+  virtualenv venv_specification
+
+Note: Make sure you are creating a Python 3 virtual environment, if required use the -p flag to set the Python interpreter (for example `virtualenv -p /absolute/path/to/python venv_specification`).
+
+Then activate the virtual environment and install the required packages::
```

Comment:
"install" is US spelling ... UK is "instal" (yeah, it looks weird ...)

Reply:
Can't believe me of all people is using American spelling urgh!

Reply:
I think it's just this **one** word ... it looks so strange ...